main.py

```diff
@@ -1,7 +1,30 @@
+# from venv import create
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    for i in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=i)
+    actors = ["George Klooney",
+              "Kianu Reaves",
```

Comment:
There are typos in the actor names 'George Klooney' and 'Kianu Reaves'. These are corrected later in the code, but it's better to input the correct names initially.